backend.go

```diff
@@ -115,6 +147,27 @@ func (b *kubeAuthBackend) loadConfig(ctx context.Context, s logical.Storage) (*k
 	if config == nil {
 		return nil, errors.New("could not load backend configuration")
 	}
+
+	// Nothing more to do if loading local CA cert and JWT token is disabled.
+	if config.DisableLocalCAJwt {
+		return config, nil
+	}
+
+	// Read local JWT token unless it was not stored in config.
+	if config.TokenReviewerJWT == "" {
+		config.TokenReviewerJWT, _ = b.localSATokenReader.ReadFile()
+		// Ignore error: make best effort trying to load local JWT,
+		// otherwise the JWT submitted in login payload will be used.
```

Comment:
One final thought from me, maybe worth adding a debug log for this, just to make it easier to understand in the rare instance that something does go wrong reading the token?

```suggestion
		config.TokenReviewerJWT, err = b.localSATokenReader.ReadFile()
		if err != nil {
			// Ignore error: make best effort trying to load local JWT,
			// otherwise the JWT submitted in login payload will be used.
			b.Logger().Debug("failed to read local service account token, will use client token", "error", err)
		}
```

Reply:
Fixed!